com.unity.render-pipelines.high-definition/Runtime/Material/Lit/LitProperties.hlsl

```diff
@@ -293,13 +293,13 @@ float _TessellationObjectScale;
 float _TessellationTilingScale;
 #endif
 
+CBUFFER_END
+
```

Comment:
TODO : Do this for all the other files including those IDs in the PerMaterial cbuffer